modules/adyoulikeBidAdapter.js

```diff
@@ -44,7 +68,13 @@ export const spec = {
         accumulator[bid.bidId].Width = size.width;
         accumulator[bid.bidId].Height = size.height;
         accumulator[bid.bidId].AvailableSizes = sizesArray.join(',');
-        if (bid.mediaTypes && bid.mediaTypes.native) accumulator[bid.bidId].Native = bid.mediaTypes.native;
+        if (bid.mediaTypes && bid.mediaTypes.native) {
+          let nativeReq = bid.mediaTypes.native;
+          if (nativeReq.type === 'image') {
+            nativeReq = NATIVE_IMAGE;
+          }
+          accumulator[bid.bidId].Native = nativeReq;
+        }
```

Comment:
@guiann So when a pub defines a "short native config type" will it always only have the single key `type` ?

Is it possible that a pub would want to use the short type, but also be able to override the default fields?

Maybe instead of 

```js
nativeReq = NATIVE_IMAGE;
```

We merge the pub provided `bid.mediaTypes.native` onto the NATIVE_IMAGE default?

Reply:
I was thinking if a publisher uses the shorthand they won't want to spend time with fine tuning...  but  yes we can handle this case.  I'll do the merge of the two objects.

Again, do every adapters have to handle this case by themselves ? Unfolding shorthand formats should be a more global treatment, don't you think?

Reply:
change is done.

Reply:
Yeah, agrees we should handle shorthands in core prebid

Reply:
You can upvote the issue I've opened   here :
https://github.com/prebid/Prebid.js/issues/6389

Thank you.